overview/content-types/shell-code.nix

```diff
@@ -23,12 +23,33 @@ in
       type = with types; functionTo str;
       readOnly = true;
       default =
-        self: with lib; ''
-          <pre class="shell"><code>${trim ''
-            ${self.prompt} ${self.input}
-            ${optionalString (!isNull self.output) self.output}
-          ''}</code></pre>
-        '';
+        self:
+        let
+          shell = input: ''
+            <pre class="shell"><code>${lib.trim ''
+              ${self.prompt} ${input}
```

Comment:
Why not

```suggestion
              ${self.prompt} ${self.input}
```

Reply:
Because currently `self.input` can be a list and that wouldn't work like this.